Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openresty/openresty:alpine-fat
+
+# Would be awesome to see this use docker-compose and build a fully
+# functional test environment
+
+COPY . /build
+
+WORKDIR /build
+RUN apk add --no-cache openssl openssl-dev git
+RUN make dev && make test
+
```

Comment:
Please remove unrelated changes from this PR, such as this Dockerfile and the Makefile rule added below, along with the README change. Thank you!